strongbox-data-service/src/main/resources/etc/conf/db_EMBEDDED.yaml

```
@@ -0,0 +1,7 @@
+strongbox:
+    db:
+      janusgraph:
+        config-location: ${strongbox.home}/etc/conf/janusgraph-cassandra.properties
+      cassandra:
+        config-locatoion: file:${strongbox.home}/etc/conf/cassandra.yaml
```

There is a typo here:
```suggestion
        config-location: file:${strongbox.home}/etc/conf/cassandra.yaml
```

Also, if I were a user, I'd expect all of the `config-location` fields to work the same way and would not set the `file:` prefix. As far as I remember, the CassandraDaemon class allowed you to use `classpath:`, but I can't think of a use case right now in which we'd be using `classpath:` instead of `file:`? 

I think we should automatically add the `file:` prefix to the string in the backend part and if in the future we need to - we can make it more robust and allow for `classpath:`. What do you think?

---

I thought that with protocol prefix it would be more flexible, so as you mentioned it could be `classpath` or `jar` protocols. For example initially I've used `jar` protocol at `strongbox-db-import` project. Probably it would be better to leave this "feature", so maybe you will decide to use `http` in the future :)

---

BTW, the `classpath` is not possible, the only similar option is `jar`.

---

`janusgraph.config-location` is now accept URL as agreed.